components/director/Makefile

```diff
@@ -43,6 +43,8 @@ deploy-on-k3d: build-for-k3d
 	kubectl patch -n $(NAMESPACE) cronjob/compass-ord-aggregator -p '{"spec":{"jobTemplate":{"spec":{"template":{"spec":{"containers":[{"name":"aggregator","imagePullPolicy":"Always"}]}}}}}}'
 	kubectl set image -n $(NAMESPACE) deployment/compass-tenant-fetcher tenant-fetcher=k3d-kyma-registry:5001/compass-director:latest
 	kubectl rollout restart -n $(NAMESPACE) deployment/compass-tenant-fetcher
+	kubectl set image -n $(NAMESPACE) deployment/compass-tenant-fetcher tenant-fetcher=k3d-kyma-registry:5001/compass-director:latest
```

Comment:
Why this is added for a second time?

Reply:
It's a mistake. Probably a bad rebase. Fixed.